Source/WebCore/css/CSSSelectorList.cpp

```diff
@@ -70,7 +70,6 @@ CSSSelectorList::CSSSelectorList(Vector<std::unique_ptr<CSSParserSelector>>&& se
             if (current != first)
                 m_selectorArray[arrayIndex].setNotFirstInTagHistory();
             current = current->tagHistory();
-            ASSERT(!m_selectorArray[arrayIndex].isLastInSelectorList());
```

Comment:
This assert is a bit weird (it works only because the default ctor write `false` in this field, and it assumes this field has never been modified). It should actually be a `setNotLastInSelectorList()` (which is overwritten for the last selector in the last line of the function) because we don't care about what is there already, we just need it to be false for all but the last one after construction.

I've changed it by keeping the assert except for the last element.